src/lib/y2storage/guided_proposal.rb

```diff
@@ -212,8 +213,7 @@ def target_sizes
       [:desired, :min]
     end
 
-    # Copy of the original settings including some calculated and necessary
-    # values (mainly candidate_devices), in case they were not present
+    # Settings used by each attempt of proposal
```

Comment:
I would extend the documentation here by saying that this "populated settings" are intended to be populated during the process of making a proposal, for example, by setting the candidate devices or the root device when it is not given in the original settings.